### PR TITLE
Updated SLS application version to 2.00.0 in 3.0.2

### PR DESCRIPTION
**Pull request by shunr-hpe**
### Summary and Scope

1. Removed encrypted dumpstate and loadstate capabilities
2. Updated the chart tests to run the new functional tests

### Issues and Related PRs

LIST AND CHARACTERIZE RELATIONSHIP TO JIRA ISSUES AND OTHER PULL REQUESTS. BE SURE LIST DEPENDENCIES.

* Resolves CASMHMS-5771, CASMHMS-5856

### Testing

Tested on:

* mug
* local docker compose environment

Were the install/upgrade based validation checks/tests run?(goss tests/install-validation doc)
Were continuous integration tests run? Y
Was an Upgrade tested? Y
Was a Downgrade tested? Y
Tested on:

### Risks and Mitigations

Low

### CODE REVIEW

File: charts/v3.0/cray-hms-sls/Chart.yaml [hunk 1]
```diff
@@ -13,4 +13,4 @@ dependencies:
     repository: https://artifactory.algol60.net/artifactory/csm-helm-charts
 annotations:
   artifacthub.io/license: MIT
-appVersion: 1.28.0
+appVersion: 2.00.0
```
Comment (rsjostrand-hpe) [inline charts/v3.0/cray-hms-sls/Chart.yaml]: See comment in the other PR: https://github.com/Cray-HPE/hms-sls/pull/62#discussion_r1042414149

**Comment by schooler-hpe** [comment]
Two things:

1. The chart changes for the test updates look good. Did you try deploying the new chart and running the SLS tests via Helm on a real system? I think it is important to do that testing for this change.
2. Can you add something to the CHANGELOG about updating to the new test framework? For FAS I said:
> - Updated FAS CT tests to use hms-test:4.0.0 image for HMTH

**Comment by shunr-hpe** [comment]
> Two things:
> 
> 1. The chart changes for the test updates look good. Did you try deploying the new chart and running the SLS tests via Helm on a real system? I think it is important to do that testing for this change.
> 2. Can you add something to the CHANGELOG about updating to the new test framework? For FAS I said:
> 
> > * Updated FAS CT tests to use hms-test:4.0.0 image for HMTH

I tested on hardware and the logs of the test pod looked like it was running the new tests. I used these manifest values when deploying.
```
apiVersion: manifests/v1beta1
metadata:
  name: sls
spec:
  charts:
  - name: cray-hms-sls
    namespace: services
    version: 3.0.2-20221206212121+9a260f6
    values:
      image:
        repository: artifactory.algol60.net/csm-docker/unstable/cray-sls
      tests:
        image:
          repository: artifactory.algol60.net/csm-docker/unstable/cray-sls-hmth-test
      cray-service:
        containers:
          cray-sls:
            image:
              repository: artifactory.algol60.net/csm-docker/unstable/cray-sls
      global:
        appVersion: 2.00.0-20221205160659.401ec17
        testVersion: 2.00.0-20221205160703.401ec17
```

**Review by schooler-hpe — APPROVED** [review]

**Comment by shunr-hpe** [comment]
Logs from smoke tests
```
ncn-m001:~/shunr/sls # kubectl -n services logs -f cray-hms-sls-test-smoke-xh7fz
Running smoke tests...
2022-12-08 23:22:24,401 suite_name: sls-smoke-tests
2022-12-08 23:22:24,401 file_path: smoke.json
2022-12-08 23:22:24,401 Running sls-smoke-tests...
2022-12-08 23:22:24,401 default_url: http://cray-sls
2022-12-08 23:22:24,402 override_url: http://cray-sls
2022-12-08 23:22:24,402 configured_url: http://cray-sls/
2022-12-08 23:22:24,402 Testing {"path": "v1/version", "expected_status_code": 200, "method": "GET", "body": null, "headers": {}, "url": "http://cray-sls/v1/version"}
2022-12-08 23:22:24,441 Starting new HTTP connection (1): cray-sls:80
2022-12-08 23:22:24,485 http://cray-sls:80 "GET /v1/version HTTP/1.1" 200 58
2022-12-08 23:22:24,486 Testing {"path": "v1/health", "expected_status_code": 200, "method": "GET", "body": null, "headers": {}, "url": "http://cray-sls/v1/health"}
2022-12-08 23:22:24,526 Starting new HTTP connection (1): cray-sls:80
2022-12-08 23:22:24,532 http://cray-sls:80 "GET /v1/health HTTP/1.1" 200 46
2022-12-08 23:22:24,533 Testing {"path": "v1/liveness", "expected_status_code": 204, "method": "GET", "body": null, "headers": {}, "url": "http://cray-sls/v1/liveness"}
2022-12-08 23:22:24,572 Starting new HTTP connection (1): cray-sls:80
2022-12-08 23:22:24,601 http://cray-sls:80 "GET /v1/liveness HTTP/1.1" 204 0
2022-12-08 23:22:24,603 Testing {"path": "v1/readiness", "expected_status_code": 204, "method": "GET", "body": null, "headers": {}, "url": "http://cray-sls/v1/readiness"}
2022-12-08 23:22:24,643 Starting new HTTP connection (1): cray-sls:80
2022-12-08 23:22:24,650 http://cray-sls:80 "GET /v1/readiness HTTP/1.1" 204 0
2022-12-08 23:22:24,650 Testing {"path": "v1/hardware", "expected_status_code": 200, "method": "GET", "body": null, "headers": {}, "url": "http://cray-sls/v1/hardware"}
2022-12-08 23:22:24,688 Starting new HTTP connection (1): cray-sls:80
2022-12-08 23:22:24,710 http://cray-sls:80 "GET /v1/hardware HTTP/1.1" 200 None
2022-12-08 23:22:24,712 Testing {"path": "v1/networks", "expected_status_code": 200, "method": "GET", "body": null, "headers": {}, "url": "http://cray-sls/v1/networks"}
2022-12-08 23:22:24,750 Starting new HTTP connection (1): cray-sls:80
2022-12-08 23:22:24,759 http://cray-sls:80 "GET /v1/networks HTTP/1.1" 200 None
2022-12-08 23:22:24,761 Testing {"path": "v1/dumpstate", "expected_status_code": 200, "method": "GET", "body": null, "headers": {}, "url": "http://cray-sls/v1/dumpstate"}
2022-12-08 23:22:24,801 Starting new HTTP connection (1): cray-sls:80
2022-12-08 23:22:24,813 http://cray-sls:80 "GET /v1/dumpstate HTTP/1.1" 200 None
2022-12-08 23:22:24,815 MAIN_ERRORS = 0
2022-12-08 23:22:24,815 PASS: sls-smoke-tests
2022-12-08 23:22:24,815 passed!
2022-12-08 23:22:24,815 PASS: sls-smoke-tests passed!
OK
```

**Comment by shunr-hpe** [comment]
logs from the functional tests
```
ncn-m001:~/shunr/sls # kubectl -n services logs -f cray-hms-sls-test-functional-96r2l
Running tavern tests...
============================= test session starts ==============================
platform linux -- Python 3.9.15, pytest-7.1.2, pluggy-1.0.0 -- /usr/bin/python3
cachedir: .pytest_cache 
rootdir: /src/app, configfile: pytest.ini
plugins: tavern-1.23.1
collecting ... collected 6 items

api/1-non-disruptive/test_dumpstate.tavern.yaml::Verify the dumpstate resource PASSED [ 16%]
api/1-non-disruptive/test_hardware.tavern.yaml::Verify the hardware resource PASSED [ 33%]
api/1-non-disruptive/test_hardware.tavern.yaml::Verify the hardware resource with a specific xname PASSED [ 50%]
api/1-non-disruptive/test_health.tavern.yaml::Verify the health resource PASSED [ 66%]
api/1-non-disruptive/test_networks.tavern.yaml::Verify the networks resource PASSED [ 83%]
api/1-non-disruptive/test_service.tavern.yaml::slsAPIServiceAPIS PASSED  [100%] 

============================== 6 passed in 5.32s ===============================
2022-12-08 23:25:08,088 PASS
OK
```

**Review by rsjostrand-hpe — APPROVED** [review]

File: charts/v3.0/cray-hms-sls/Chart.yaml (hunk 1 above)
Comment (shunr-hpe) [reply charts/v3.0/cray-hms-sls/Chart.yaml]: Thanks for catching this. I fixed it.

**Pull request merged**